### ISSUE TITLE
`NOT in an emulator shell` error despite `adb shell` command working 

### ISSUE THREAD

**Issue by rew1nter** [newbit1/rootAVD#72]
Linux:
![image](https://github.com/newbit1/rootAVD/assets/64508791/ee04cb8e-9b39-4372-883c-3dd2d989833a)

**Comment by newbit1** [comment]
The find busybox routine had false positives. It turned out, that even a busybox version compiled for different
architecture can run, but does not execute the commands right, hence the issues you had.
My apologies for your inconveniences. It should be fixed now.

**Comment by rew1nter** [comment]
Hey, Thanks for the quick fix. I could root the emulator. 

But it still prints the error message, which could be misleading for others:
![image](https://github.com/newbit1/rootAVD/assets/64508791/b04b1ca5-969a-4dcf-844a-cef16582b37d)

**Comment by newbit1** [comment]
Great.

This is not an error message. It is an information where the script is right now.

**Comment by pnthai88** [comment]
> Hey, Thanks for the quick fix. I could root the emulator.
> 
> But it still prints the error message, which could be misleading for others: ![image](https://user-images.githubusercontent.com/64508791/238550084-b04b1ca5-969a-4dcf-844a-cef16582b37d.png)

It's meaning that you need to execute an adb command first or adb debugging on the emulator is running. After few retries it's working

**Comment by newbit1** [comment]
not sure how to take your concern.
Do you mean the word Test insted of Testing?

that test could be understood as a task for the user?

shall rephrase it to testing?
